File: pos-maui/Modelo/Entidad/Usuario.cs
namespace PosMaui.Modelo.Entidad;

public class Usuario
{
    public int    Id      { get; set; }
    public string Nombre  { get; set; } = "";
    public string Rol     { get; set; } = "";
    public bool   Activo  { get; set; } = true;

    public bool EsAdmin => Rol == "ADMIN";
}

File: puntoDeVenta/Controlador/src/main/java/com/empresa/pos/modelo/Usuario.java
package com.empresa.pos.modelo;

public class Usuario {
    private final String nombre;
    private final String rol; // "ADMIN", "CAJERO", "SUPERVISOR"

    public Usuario(String nombre, String rol) {
        this.nombre = nombre;
        this.rol = rol;
    }

    public String getNombre() { return nombre; }
    public String getRol()    { return rol; }
    public boolean esAdmin()  { return "ADMIN".equals(rol); }
}
